libs/fills/src/lib/fills-data-provider.ts

```diff
@@ -17,19 +17,41 @@ const FILL_FRAGMENT = gql`
     size
     buyOrder
     sellOrder
+    aggressor
     buyer {
       id
     }
     seller {
       id
     }
+    buyerFee {
+      makerFee
+      infrastructureFee
+      liquidityFee
+    }
+    sellerFee {
+      makerFee
+      infrastructureFee
+      liquidityFee
+    }
     market {
       id
+      name
       decimalPlaces
+      positionDecimalPlaces
       tradableInstrument {
         instrument {
           id
           code
+          product {
```

Comment:
question (blocking): why are these new fields needed? Is this conflicts?

Reply:
They are needed to format price values with the right decimal places and were accidentally removed. We loose some type safety when passing code through to AgGrid. @bglownia is sureing up the types in one of his PRs atm.